syntax = "proto3";

import "google/protobuf/empty.proto";

message EchoRequest {
  string msg = 1;
}

message EchoReply {
  string msg = 1;
}

service EchoService {
  rpc Echo(EchoRequest) returns (EchoReply) {}
}

service EchoProxy {
  rpc Forward1(EchoRequest) returns (EchoReply) {}
  rpc Forward2(EchoRequest) returns (EchoReply) {}
}
